spec.bs

```diff
@@ -3200,16 +3200,18 @@ The <dfn for=Navigator method>getInterestGroupAdAuctionData(|configIDL|)</dfn> m
      1. [=Queue a global task=] on the [=DOM manipulation task source=], given |global|, to
         [=reject=] |p| with a {{TypeError}}.
      1. Abort these steps.
-  1. Let |key| be a [=byte sequence=] containing one of the [[RFC9180|HPKE]] public
-     encryption keys for |config|'s [=auction data config/seller=] as determined
-     by |config|'s [=auction data config/coordinator=], or failure if no key is
+  1. Let |key|,|key id| be a [=byte sequence=] containing one of the [[RFC9180|HPKE]] public

```

Comment:
this seems broken. Key is a [=byte sequence=], not (|key|, |keyId|). Need to rewrite it a little bit

Reply:
Rephrased (pulled out to a function).